package.json

```diff
@@ -2,19 +2,27 @@
   "name": "automata",
   "version": "1.0.0",
   "scripts": {
-    "dev": "tsc-watch --onSuccess \"node dist/index.js\"",
+    "dev": "pocketbase-typegen --env --out ./util/pbtypes.ts;tsc-watch --onSuccess \"node dist/index.js --dev\"",
     "build": "tsc"
   },
   "type": "module",
   "dependencies": {
+    "axios": "^1.5.0",
```

Comment:
switch to using vanilla fetch, which is native in version 18

Reply:
even though I like fetch, axios is simpler imo so I'd keep it